instrumentation/opentelemetry-instrumentation-asyncpg/src/opentelemetry/instrumentation/asyncpg/__init__.py

```diff
@@ -150,8 +150,8 @@ def _uninstrument(self, **__):
 
     async def _do_execute(self, func, instance, args, kwargs):
         exception = None
-        params = getattr(instance, "_params", {})
```

Comment:
Would we still want to keep the default to prevent other AttributeError?

Reply:
Yup, fixed!

Reply:
Thanks! We should still use the original default of `{}` instead of `None`, else the `getattr` call will give AttributeError for NoneType object if there's no `_params`.

Reply:
Not really - `getattr`works just fine on `None`:
```py
>>> getattr(None, '_params', 'bar')
'bar'
```

That means that if there is no `_params`, we would have `params = None`. After that we do `getattr(params, "database", "postgresql")` which is evaluated to `getattr(None, "database", "postgresql")` which finally evaluates to `"postgresql"`.

Reply:
Yes, you are correct 👍